app/models/user_course.rb

```diff
@@ -156,7 +156,7 @@ def update_exp_and_level
     self.exp = self.exp >= 0 ? self.exp : 0
 
     unless self.level
-      self.level = self.course.levels.find_by_level(0)
+      self.level = course.levels.find_by_level(0) || course.levels.first
```

Comment:
here should be level 0. we just need to populate those course without level 0. no need to change here.